site/pages/create-consequence-services/__snapshots__/create-consequence-services.test.tsx.snap

```diff
@@ -626,12 +626,14 @@ exports[`pages > CreateConsequenceServices > should render correctly with errors
                 <div
                   aria-describedby="removeFromJourneyPlanners-error"
                   className="govuk-radios govuk-radios--error"
+                  data-module="govuk-radios"
```

Comment:
When I choose daily and then enter all the details besides the start time then press submit the radios become disabled and default to doesn't repeat rather than what I entered for weekly - is that intended?

Reply:
I think it is the ordering I select weekly or daily first then enter a value then select no end date and time so then it doesn't revert back to doesn't repeat rather it just disabled the radios on the value I entered

Reply:
done